.github/workflows/ci.yml

```
@@ -44,6 +44,32 @@ jobs:
       - name: before_cache_script
         run: rm -rf $CARGO_HOME/registry/index
 
+  macos-silicon:
```

I suggest renaming this new job to `macos-aarch64`, and since we want to keep both these 2 CIs, we should rename the old `macos` job to `macos-x86_64`.

```suggestion
  macos-aarch64:
```

---

> we should rename the old macos job to macos-x86_64.

It would be better if we rename the old `macos` job to `macos-x86_64` for symmetry reasons.